gm8emulator/src/gml/kernel.rs

```diff
@@ -472,19 +472,53 @@ impl Game {
         expect_args!(args, [int]).map(|c| (c / 256 / 256) % 256).map(Value::from)
     }
 
-    pub fn color_get_hue(&mut self, _context: &mut Context, _args: &[Value]) -> gml::Result<Value> {
-        // Expected arg count: 1
-        unimplemented!("Called unimplemented kernel function color_get_hue")
+    pub fn color_get_hue(&mut self, _context: &mut Context, args: &[Value]) -> gml::Result<Value> {
+        let c = expect_args!(args, [int])?;
+        let (h, _s, _v) = self.rgb_to_hsv(c);
```

Comment:
Values that don't matter can be ignored with _, such as:
```rust
let (h, _, _) = self.rgb_to_hsv(c);
```

Reply:
Ohhh, I thought the `_` was a prefix for unused vars.
Changed in latest commit

Reply:
It is! `_` is a special value for vars that should be computed, but you want to ignore the result. It actually immediately drops the result!